stdlib/public/runtime/DynamicCast.cpp

```diff
@@ -752,6 +752,12 @@ struct ObjCBridgeMemo {
 };
 #endif
 
+const Metadata *getNSStringMetadata() {
```

Comment:
This should go in stdlib/public/runtime/SwiftObject.mm with the other getNSXyzMetadata() methods.

Reply:
In fact, I would suggest also putting a `getNSStringHashableConformance()` there as well.